GrievanceManagement/src/main/java/com/gms/serviceImpl/TicketServiceImpl.java

```diff
@@ -73,61 +91,105 @@ public Ticket saveTicket(final TicketSaveInDTO ticketSaveInDTO) {
         return ticketRepository.save(ticket);
     }
 
+    /**
+     * This is @getAllTicket for getting list of @TicketTableOutDTO.
+     */
     @Override
-    public List<TicketTableOutDTO> getAllTicket() {
-        List<TicketTableOutDTO> ticketTableOutDTOs = ticketRepository.findAllTicket();
-        if (ticketTableOutDTOs.size() == 0) {
-            LOGGER.warn("[TicketServiceImpl]: Empty list of ticket");
-            throw new TicketNotFoundException("There is no ticket");
-        }
-        LOGGER.info("[TicketServiceImpl]: Sorting list of ticket basis of status");
-        List<TicketTableOutDTO> sortedOnStatus = ticketTableOutDTOs.stream().sorted((ticket1, ticket2) -> {
-            if (ticket1.getStatus().equals(ticket2.getStatus())) {
-                return ticket2.getLastUpdationTime().compareTo(ticket1.getLastUpdationTime());
-            } else if (ticket1.getStatus().equals(Status.OPEN)) {
-                return -1;
-            } else if (ticket1.getStatus().equals(Status.BEING_ADDRESSED)) {
-                return ticket2.getStatus().equals(Status.OPEN) ? 1 : -1;
-            } else {
-                return 1;
+    public List<TicketTableOutDTO> getAllTicket(final Long userId, final Boolean myTicket) {
+        List<TicketTableOutDTO> ticketTableOutDTOs;
+        User user = userRepository.findById(userId).orElseThrow(
+                () -> new NotFoundException("UserId not found"));
+        if (myTicket && user.getTicket().size() == 0) {
```

Comment:
use equals method instead of == every where and create constants for integer values